Tween/DelegateAnimation.cs

```diff
@@ -3,13 +3,33 @@
 
 namespace DUCK.Tween
 {
+	public abstract class DelegateAnimation : AbstractAnimation
```

Comment:
I kept this non-generic version of Delegate Animation just in case you ever want to cast your animation to an unknown typed Delegate Animation.

Reply:
I wonder if it could implement `IAnimationPlaybackControl`.  it's abstract so may just force you to add abstract function signature. it's probably not needed but i guess it's more protetion against subclassing this and trying to abuse the system

Reply:
Done.